dpp/src/state_transition/abstract_state_transition_identity_signed.rs

```diff
@@ -75,7 +74,11 @@ where
         }
     }
 
-    fn verify_signature(&self, public_key: &IdentityPublicKey) -> Result<(), ProtocolError> {
+    fn verify_signature(
+        &self,
+        public_key: &IdentityPublicKey,
+        bls: &impl BlsModule,
```

Comment:
I'm just wondering if there is a way to avoid this dependency. But honestly, I don't have a better idea. Let it be that way so far.

Reply:
Yeah, I've though how to do it the other way, but if we want to pass an external adapter for different environment, I didn't came up with anything better